dataprofiler/labelers/column_name_model.py

```diff
@@ -0,0 +1,232 @@
+            value = parameters[param]
+            if param == "false_positive_dict" and ( 
+                not isinstance(value, list)
+                or 'attribute' not in value[0].keys()
+            ):
+                errors.append(
+                    "`{}` must be a list of dictionaries with at least 'attribute' as the key".format(param)
+                )
+            elif param == "true_positive_dict" and (
+                not isinstance(value, list)
+                or not isinstance(value[0], dict)
+                or not isinstance(value[1]['attribute'], str)
+                or not isinstance(value[1]['label'], str)
+            ):
+                errors.append(
+                    """`{}` must be a list of dictionaries each with the following
+                    two keys: 'attribute' and 'label'""".format(param)
+                )
+            elif param not in list_of_accepted_parameters:
+                errors.append("`{}` is not an accepted parameter.".format(param))
+        if errors:
+            raise ValueError("\n".join(errors))
+
+    def _compare_negative(self, list_of_column_names, check_values_dict, negative_threshold):
+        """Filter out column name examples that are false positives"""
+        scores = self._model(
+                list_of_column_names,
+                check_values_dict,
+                self._make_lower_case(),
+                fuzz.token_sort_ratio,
+                include_label=False)
+
+        list_of_column_names_filtered = []
+        for i in range(len(list_of_column_names)):
+            if scores[i][0] < negative_threshold:
+                list_of_column_names_filtered.append(list_of_column_names[i])
+
+        return list_of_column_names_filtered
+
+    def _compare_positive(self, list_of_column_names, check_values_dict, positive_threshold, include_label):
+        """Calculate similarity scores between list of column names and true positive examples"""
+        
+        scores = self._model(
+            list_of_column_names,
+            check_values_dict,
+            self._make_lower_case(),
+            fuzz.token_sort_ratio,
+            include_label=include_label,
+        )
+
+        output_dictionary = {}
+        for i in range(len(list_of_column_names)):
+            if scores[i][0] > positive_threshold:
+                output_dictionary[list_of_column_names[i]] = {}
+                output_dictionary[list_of_column_names[i]]['prediction'] = \
+                    check_values_dict[scores[i]]['label']
+                output_dictionary[list_of_column_names[i]]['Similarity Score'] = scores[i][0]
+
+        return output_dictionary
+
+    def _construct_model(self):
+        pass
+
+    def _reconstruct_model(self):
+        pass
+
+    def _need_to_reconstruct_model(self):
+        pass
+
+    def reset_weights(self):
+        """Reset weights."""
+        pass
+
+    def _make_lower_case(str, **kwargs):
+        return str.lower()
```

Comment:
should it be `str.lower()` or `str.lower`?

Reply:
```In [1]: str.lower
Out[1]: <function str.lower()>
```
should return a `str` object type... so think we actually do want to call the `lower()` as a function call